gst/nnstreamer/tensor_converter/README.md

```diff
@@ -26,13 +33,14 @@ From higher priority
 
 ## Sink Pads
 
-One "Always" sink pad exists. The capability of sink pad is ```video/x-raw```, ```audio/x-raw```, ```text/x-raw``` and ```application/octet-stream```.
+One "Always" sink pad exists. The capability of sink pad is ```video/x-raw```, ```audio/x-raw```, ```text/x-raw```, ```application/octet-stream``` and ```other/tensors-flexible```.
```

Comment:
A, B, C ==> A, B, and C (a comma is missing)

Reply:
updated